sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/FormRecognizerLocale.java

```diff
@@ -10,6 +10,13 @@
  * Defines values for FormRecognizerLocale.
  */
 public final class FormRecognizerLocale extends ExpandableStringEnum<FormRecognizerLocale> {
+
+    /**
+     * Creates a FormRecognizerLocale object.
+     */
+    public FormRecognizerLocale() {
```

Comment:
Since this is an `ExpandableStringEnum` this will need to be marked as deprecated and tell users to use the factory method.